release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -175,6 +179,43 @@ spec:
   storageClassName: snapshot-promoter
 ----
 
+[[ocp-310-new-storage-provisioners]]
+==== New Storage Provisioners (Technology Preview)
+
+New Storage Provisioners (CephFS) are currently in
```

Comment:
@tkatarki - should we be removing this from the release notes as per your email?

@ahardin-rh @aheslin